packages/@uppy/companion-client/types/index.d.ts

```diff
@@ -1,48 +1,48 @@
 export interface RequestClientOptions {
-  companionUrl: string;
-  companionHeaders?: object;
+  companionUrl: string
+  companionHeaders?: object
   /**
    * Deprecated, use `companionHeaders` instead.
    */
-  serverHeaders?: object;
+  serverHeaders?: object
 }
 
```

Comment:
we should also add token Storage interface like so 

```typescript
export interface TokenStorage {
  setItem: (key: string, value: string) => Promise<any> // or <void> not sure
  getItem: (key: string) => Promise<string>
  removeItem: (key: string) => Promise<any> // or <void> not sure
}
```

it will be used in `packages/@uppy/dropbox/types/index.d.ts` and other providers

Reply:
did this get added in a different way somewhere? 🤔

Reply:
comitted but didn't push :upside_down_face: should be there now!